moduleroot/.travis.yml.erb

```
@@ -88,7 +88,18 @@ branches:
 <%   end -%>
 <% end -%>
 notifications:
-  email: false
+<% notifications_arr = ((@configs['notifications'].to_a || []) - (@configs['remove_notifications'].to_a || [])) -%>
```

Does this need to guard against `@configs['notifications']` possibly being `nil`? I realize it has a default, but someone maybe could override it to `nil`?

---

Was there a situation where this value was not an array but could be safely coerced into an array? If not, we can just drop to:

```suggestion
<% notifications_arr = ((@configs['notifications'] || []) - (@configs['remove_notifications'] || [])) -%>
```

---

@scotje Notifications is a hash, but in order to provide a "difference" between the `@config['notifications']` and `@config['remove_notifications']` i converted to an array to make this difference logic easier.